<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_showNormal"/>
    <addaction name="action_showFullScreen"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Donate"/>
    <addaction name="separator"/>
    <addaction name="action_Issues"/>
    <addaction name="action_Chat"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dock_gcode">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>G-Code</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGCodeEditor" name="gcode"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>524287</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Bash</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPlainTextEdit" name="command">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="plainText">
        <string>echo &quot;Hello, World!&quot; | hf2gcode -q</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Quit"/>
   <addaction name="separator"/>
   <addaction name="action_showFullScreen"/>
   <addaction name="action_showNormal"/>
   <addaction name="separator"/>
   <addaction name="action_AutoZoom"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>Standard Error</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPlainTextEdit" name="stderror"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="priority">
    <enum>QAction::HighPriority</enum>
   </property>
  </action>
  <action name="action_showFullScreen">
   <property name="text">
    <string>&amp;Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="action_showNormal">
   <property name="text">
    <string>&amp;Normal</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Donate">
   <property name="text">
    <string>&amp;Donate</string>
   </property>
  </action>
  <action name="action_AutoZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Zoom</string>
   </property>
   <property name="toolTip">
    <string>Auto Zoom</string>
   </property>
  </action>
  <action name="action_Issues">
   <property name="text">
    <string>&amp;Issues</string>
   </property>
  </action>
  <action name="action_Wiki">
   <property name="text">
    <string>&amp;Wiki</string>
   </property>
  </action>
  <action name="action_Chat">
   <property name="text">
    <string>&amp;Chat</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QGCodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>QGCodeEditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_showFullScreen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showFullScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_showNormal</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showNormal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
